scripts/minikube-minishift-setup-env.sh

```diff
@@ -39,6 +39,62 @@ setup_kubeconfig() {
     fi
 }
 
+setup_minikube_developer() {
+    pwd=`pwd`
+    certdir=`mktemp -d`
+    cd $certdir
+    shout "Starting to create minikube developer"
+    openssl genrsa -out developer.key 2048
+    openssl req -new -key developer.key -out developer.csr -subj "/CN=developer/O=minikube"
+    openssl x509 -req -in developer.csr -CA ~/.minikube/ca.crt -CAkey ~/.minikube/ca.key -CAcreateserial -out developer.crt -days 500
+    kubectl config set-credentials developer --client-certificate=developer.crt --client-key=developer.key
+    kubectl config set-context developer-minikube --cluster=minikube --user=developer
+    kubectl create -f - <<EOF
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Please comment that the intent of this role is to remove access to the CRD resource